src/sys/unix.rs

```diff
@@ -6,7 +6,7 @@
 // option. This file may not be copied, modified, or distributed
 // except according to those terms.
 
-use std::io::{Read, Write};
+use std::io::{IoSlice, IoSliceMut, Read, Write};
```

Comment:
Wouldn't this need `#[cfg(not(target_os = "redox"))]`?

Reply:
Good catch, updated :)